src/pudl_archiver/archivers/ferc/ferceqr.py

```diff
@@ -0,0 +1,68 @@
+"""Download FERC EQR data."""
+from pathlib import Path
+
+from pudl_archiver.archivers.classes import (
+    AbstractDatasetArchiver,
+    ArchiveAwaitable,
+    ResourceInfo,
+)
+
+BASE_URL = "https://eqrreportviewer.ferc.gov/"
+
+
+class FercEQRArchiver(AbstractDatasetArchiver):
+    """FERC EQR archiver."""
+
+    name = "ferceqr"
+
+    async def get_resources(self) -> ArchiveAwaitable:
+        """Download FERC EQR resources."""
+        # Get non-transaction data (pre-2013)
+        yield self.get_bulk_csv()
+
+        # Get 2002-2013 annual transaction data
+        for year in range(2002, 2014):
+            yield self.get_year_dbf(year)
+
+        # Get quarterly EQR data
+        for year in range(2014, 2023):
+            for quarter in range(1, 5):
+                yield self.get_year_dbf(year, quarter)
+
+    async def get_bulk_csv(self) -> tuple[Path, dict]:
+        """Download all 2002-2013 non-transaction data."""
+        url = "https://eqrdds.ferc.gov/eqrdbdownloads/eqr_nontransaction.zip"
+        download_path = self.download_directory / "ferceqr_nontrans.zip"
+        await self.download_zipfile(url, download_path)
+
+        return ResourceInfo(
+            local_path=download_path, partitions={"data_type": "non-transaction"}
+        )
+
+    async def get_year_dbf(
```

Comment:
non-blocking: It might be a little cleaner to split this into two separate functions, one for yearly and one for quarterly - then you aren't, like... checking if certain params were `None`, you can just call the right function. IDK if that feels better to you.

Reply:
Makes sense to me, implemented.